helmclient.go

```diff
@@ -118,7 +119,9 @@ func New(config Config) (*Client, error) {
 }
 
 // DeleteRelease uninstalls a chart given its release name.
-func (c *Client) DeleteRelease(ctx context.Context, releaseName string, options ...helmclient.DeleteOption) error {
+func (c *Client) DeleteRelease(ctx context.Context, releaseName string, options ...helmclient.DeleteOption) (err error) {
+	eventName := "delete_release"
```

Comment:
I'm a quite python fan, but not sure what is `__func__` equivalent for golang.

Reply:
Should it be const strings?

Reply:
I'm not sure either. Maybe the `reflect` package? I'd suggest researching it if you want to use this.